File: PS/gidoong/codewars/8kyu/Quarter of the year.go
package kata

func QuarterOf1(month int) (res int) {
  switch month {
    case 1, 2, 3 : res=1
    case 4, 5, 6 : res=2
    case 7, 8, 9 : res=3
    case 10,11,12: res=4
  }
  return
}

func QuarterOf2(month int) int {
  if month <= 3 {
    return 1
  }
  if month <= 6 && month > 3 {
    return 2
  }
  if month <= 9 && month > 6 {
    return 3
  }
  if month <= 12 && month > 9 {
    return 4
  }
  return 0
}

File: PS/gidoong/codewars/8kyu/Quarter of the year.py
def quarter_of1(month):
    return (month+2) // 3

from math import ceil
def quarter_of2(month):
    return ceil(month/3)

def quarter_of(month):
    if month in range(1, 4):
        return 1
    if month in range(4, 7):
        return 2
    if month in range(7, 10):
        return 3
    if month in range(10, 13):
        return 4

